### PR TITLE
TCM-115 | Build dummy workout screen

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy workout screen.
- [x] Added workout to test utility.

### Issue

[TCM-115](https://hyfnla.atlassian.net/browse/TCM-115)

### Video/Screenshot

| State Book | State Cancel |State Join |
|--------|------|------|
|workout_state_book|workout_state_cancel|workout_state_join|

### CODE REVIEW

**Comment by onodude** [comment]
lets resolve the conflicts and match the latests specs

**Comment by onodude** [comment]
lets resolve the conflicts and match the latests specs

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        viewBinding.btnBack.setOnClickListener { viewModel.doTapBack() }
+    }
+
+    private val handleTapBack = Observer<Void> {
+        NavigationManager.shared.dismiss(this)
+    }
+
+    private val handleLoadView = Observer<BookingViewState> {
+        viewBinding.apply {
+            btnPrimary.setBackgroundColor(
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt:34]: Can just do 
```
btnPrimary.bgColor(it.buttonBackgroundColor)
btnPrimary.textColor(it.buttonTitleColor)

File: app/src/main/res/layout/fragment_workout.xml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                        android:src="@drawable/ic_btn_back_white"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+
+                </com.trx.consumer.common.CommonView>
+
+                <com.trx.consumer.common.CommonImageView
+                    android:id="@+id/imgHeader"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_workout.xml:53]: This image should take up the nav bar space as well

File: app/src/main/res/layout/fragment_workout.xml [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                    android:layout_width="match_parent"
+                    android:layout_height="0dp"
+                    android:scaleType="fitCenter"
+                    app:layout_constraintBottom_toBottomOf="@+id/viewHeader"
+                    app:layout_constraintTop_toBottomOf="@+id/viewNavBar" />
+
+                <com.trx.consumer.common.CommonLabel
+                    android:id="@+id/lblDate"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_workout.xml:61]: This, `lblTitle`, and `lblSubtitles`, should have a margin start of 15dp

File: app/src/main/res/layout/fragment_workout.xml [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+                    android:textColor="@color/black"
+                    android:textSize="16sp"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toStartOf="@+id/btnTrainerProfile"
+                    app:layout_constraintStart_toEndOf="@+id/imgTrainerProfile"
+                    app:layout_constraintTop_toTopOf="parent"
+                    tools:text="Jamie Meneses" />
+
+                <com.trx.consumer.common.CommonButton
+                    android:id="@+id/btnTrainerProfile"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_workout.xml:138]: Needs a `border`

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt (hunk 1 above)
Comment (halilyesilyurt) [reply app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt]: completed.

File: app/src/main/res/layout/fragment_workout.xml (hunk 2 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_workout.xml:53]: completed.

File: app/src/main/res/layout/fragment_workout.xml (hunk 3 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_workout.xml:61]: completed.

File: app/src/main/res/layout/fragment_workout.xml (hunk 4 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_workout.xml:138]: completed.

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — APPROVED** [review]

**Pull request merged**